server/scripts/20220722000000-backfill-subscriptions.ts

```diff
@@ -0,0 +1,65 @@
+import "./bootstrap";
+import { Subscription, Document, Event } from "@server/models";
+
+const limit = 100;
+let page = parseInt(process.argv[2], 10);
+page = Number.isNaN(page) ? 0 : page;
+
+export default async function main(exit = false) {
+  const work = async (page: number) => {
+    console.log(`Backfill subscription… page ${page}`);
+
+    // Retrieve all documents within set limit.
+    const documents = await Document.findAll({
+      limit,
+      offset: page * limit,
+      order: [["createdAt", "DESC"]],
```

Comment:
```suggestion
      order: [["createdAt", "ASC"]],
```

Better to go from old -> new otherwise documents that are created while the script is running won't get processed